src/lib.rs

```diff
@@ -14,7 +14,7 @@ lazy_static! {
     )
     .unwrap();
     static ref TIME_RE: Regex =
-        Regex::new(r"TIME:? \d{1,2}\.\d{2} (A|P)\.M\.? [–-] \d{1,2}\.\d{2} (A|P)\.M\.?").unwrap();
+        Regex::new(r"TIME:? \d{1,2}\.\d{2}\s{0,1}(A|P)\.M\.? [–-] \d{1,2}\.\d{2}\s{0,1}(A|P)\.M\.?").unwrap();
```

Comment:
Thanks for the change @ibutiti. A small suggestion though, we should cover both cases, with or without spaces, so I suggest the following changes to your PR

```suggestion
        Regex::new(r"TIME:? \d{1,2}\.\d{2}\s?{0,1}(A|P)\.M\.? [–-] \d{1,2}\.\d{2}\s?{0,1}(A|P)\.M\.?").unwrap();
```